graphql/query_resolver.go

```diff
@@ -840,10 +840,10 @@ func (r *queryResolver) MainlineCommits(ctx context.Context, options MainlineCom
 		if err != nil {
 			// This shouldn't really happen, but if it does, we should return an error and log it
 			grip.Warning(message.WrapError(err, message.Fields{
-				"message":    "Error getting most recent version",
+				"message":    "getting most recent version",
```

Comment:
You can leave this one as-is since it's a grip warning
```suggestion
				"message":    "Error getting most recent version",
```